ch3/Makefile

```
@@ -0,0 +1,10 @@
+init:
+	pip install -U pip
+	pip install -r requirements.txt
```

```suggestion
	pip install boto3==1.26.8 mlflow==1.30.0 scikit-learn==1.1.3
```
requirements.txt 를 굳이 둘 필요가 없어보이는데요. 이렇게 가면 어떨까요?

---

반영했습니다 :)